docs/conf.py

```diff
@@ -13,8 +13,6 @@
 #
 import os
 import sys
-import sphinx_rtd_theme
-import enum_tools
```

Comment:
Why do you delete it from here? Do you still get a nice description of enums?

Reply:
They were showing as unused import. I'm not sure if it changes anything in the docs, perhaps you can compare [docs built for this PR](https://starknetpy--185.org.readthedocs.build/en/185/) with [current](https://starknetpy.readthedocs.io/en/latest/index.html)

Reply:
Interesting, looks like enums are still rendered properly.